src/index.js

```diff
@@ -126,6 +126,10 @@ export function text(element, stylesheets, onUpdate, options) {
     addHref(component, element, stylesheets, onUpdate, options);
 }
 
+// Provides the date format function to use in date fields
+// in the screen.
+export const FormatDateContext = React.createContext();
```

Comment:
Very minor: I think `DateFormatContext` is a better name since it provides the date format. FormatDate sounds like an action or function which formats the date.